projects/kit/components/marker-icon/marker-icon.style.less

```diff
@@ -81,18 +81,21 @@
 
     &[data-marker-mode='link'] {
         .transition(color);
-        cursor: pointer;
         color: var(--tui-link);
         background-color: var(--tui-secondary);
 
-        &:hover {
-            color: var(--tui-link-hover);
-            background-color: var(--tui-secondary-hover);
-        }
+        a& {
```

Comment:
It's common to use `button` too, let's add it.

Reply:
fixed